code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite_goliath.dm

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
Convert this to proper autodoc format

Reply:
Sorry, could you explain what you mean here?  I just looked at what was done for the megafauna and copied that.

Reply:
Since recently we have been using autodoc. https://github.com/tgstation/tgstation/pull/45468/files heres an example

Reply:
Okay, that seems simple enough.  Done for upcoming fix.